docs/source-pytorch/advanced/training_tricks.rst

```diff
@@ -91,6 +91,31 @@ longer training time. Inspired by https://github.com/BlackHC/toma.
     # Find the batch size
     trainer.tune(model)
 
+
+You can also customize the :class:`~pytorch_lightning.callbacks.batch_size_finder.BatchSizeFinder` callback to run at different epochs. This feature is useful while fine-tuning models since
```

Comment:
same suggestion as in my other comment :)

Reply:
done!